pkg/controllers/provisioning/suite_test.go

```diff
@@ -220,6 +220,43 @@ var _ = Describe("Provisioning", func() {
 		Expect(len(nodes.Items)).To(Equal(1))
 		ExpectScheduled(ctx, env.Client, pod)
 	})
+	It("should expect terminationGracePeriod to be nil when nodepool and global value is not set", func() {
+		nodePool := test.NodePool()
+		provisioning.GlobalDefaultTerminationGracePeriod = &metav1.Duration{Duration: 98 * time.Hour}
```

Comment:
We should make sure we're resetting this to `nil` in the `BeforeEach` otherwise it leaks out of this test.

Reply:
The NodePools are getting created in each tests, we don't need to reset the value

Reply:
I added the `nil` to the `BeforeEach`